packages/api/package.json

```diff
@@ -13,7 +13,8 @@
     "prebuild": "rm -rf dist/; npm run prebuild.packageConfig",
     "prebuild.packageConfig": "node -p \"'// This file is automatically updated by the build script.\\nexport const PACKAGE_NAME = \\'' + require('./package.json').name + '\\';\\nexport const PACKAGE_VERSION = \\'' + require('./package.json').version + '\\';'\" > src/packageInfo.ts; git add src/packageInfo.ts",
     "prepack": "npm run build",
-    "test": "nyc mocha \"test/**/*.test.ts\""
+    "test": "nyc mocha $(find test -name '*.test.ts' -not -path '*/smoketest.test.ts')",
```

Comment:
Because the smoketest runs on 1k+ specs and takes a good amount of time I'm splitting it out because I only ever want to intentionally run the smoketest.